bottomnavigation/bottomnavigation/src/androidTest/java/com/microsoft/device/dualscreen/bottomnavigation/utils/SimpleBottomNavigationActivity.kt

```diff
@@ -10,6 +10,6 @@ import com.microsoft.device.dualscreen.bottomnavigation.test.R
 
 class SimpleBottomNavigationActivity : BaseTestActivity() {
     override fun getContentViewLayoutResId(): Int {
-        return R.layout.activity_simple_surfaceduo_bottomnavigation
+        return R.layout.activity_simple_bottomnavigation
     }
```

Comment:
I think we should have a few tests that use the `WM` testing artifact so we can mock easily a FoldingFeature and test that the bottom navigation is aligned to the left, to the end or spanned across displays.
Also that when we have an horizontal FoldingFeature, the bottom navigation is just shown as usual since we can not align it (because there is not a FoldingFeature separating it).